google/cloud/bigquery/_tqdm_helpers.py

```diff
@@ -106,7 +106,7 @@ def wait_for_query(
             )
             progress_bar.update(default_total)
             progress_bar.set_description(
-                "Query complete after {:0.2f}s".format(time.time() - start_time),
+                "BigQuery job ID {} successfully executed".format(query_job.job_id),
```

Comment:
Why remove the duration as opposed to combining the duration and job id?
Also, f-strings are often better than the format method for combining strings and variable values. I recommend them highly.


```suggestion
                f"BigQuery job ID {query_job.job_id} successfully executed in {time.time() - start_time:0.2f} seconds.",
```

Reply:
The duration isn't being calculated correctly, so I removed it. We can reopen a separate issue to dig into that more.